war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -166,6 +166,19 @@ var createPluginSetupWizard = function(appendTarget) {
 			title: text
 		}).tooltip('show');
 	});
+	
+	// handle clicking links that might not get highlighted due to position on the page
+	$wizard.on('click', '.nav>li>a', function(){
+		var $li = $(this).parent();
+		var activateClicked = function() {
+			if(!$li.is('.active')) {
+				$li.parent().find('>li').removeClass('active');
+				$li.addClass('active');
+			}
+		};
+		setTimeout(activateClicked, 150); // this is the scroll time
+		setTimeout(activateClicked, 250); // this should combat timing issues
```

Comment:
@recena the note says it all... at 150ms, scrolling should be completed and the element marked `active`, however if there are any hiccups or it doesn't, the actual bootstrap scrollspy might receive another scroll event and highlight something else, in which case we wait and follow up with another to hopefully handle this case